packages/api-sdk/src/medical/models/patient-settings.ts

```diff
@@ -0,0 +1,17 @@
+import { z } from "zod";
+
+const subscriptionsSchema = z.object({
+  adt: z.boolean().optional(),
+});
+
+export type PatientSubscriptions = z.infer<typeof subscriptionsSchema>;
+
+export const patientSettingsDataSchema = z.object({
+  subscriptions: subscriptionsSchema.optional(),
+});
+
+export const patientSettingsSchema = z.object({
+  settings: patientSettingsDataSchema.optional(),
+});
```

Comment:
nit: I suggest we rename `patientSettingsDataSchema` to `patientSettingsSchema` and have the...

```ts
z.object({
  settings: patientSettingsDataSchema.optional(),
})
```

...as part of the `patientCreateSchema`, like we have the `externalId`.

This allows for the schema be named according with what it represents - there's no "data" patient settings.